applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -73,32 +71,35 @@ static const uint8_t cap_adv_data[] = {
 	BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED,
 };
 
-#define AVAILABLE_SOURCE_CONTEXT                                                                   \
-	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
+#if CONFIG_BT_PACS_SNK_CONTEXT
+#define AVAILABLE_SINK_CONTEXT (CONFIG_BT_PACS_SNK_CONTEXT)
+#else
+#define AVAILABLE_SINK_CONTEXT BT_AUDIO_CONTEXT_TYPE_PROHIBITED
```

Comment:
It looks like CONFIG_BT_PACS_SNK_CONTEXT is always set in our Kconfig.defaults file

Reply:
I don't think it exists if CONFIG_BT_RX=n

Reply:
Not part of this PR anymore.